.github/workflows/proto.yml

```
@@ -40,3 +44,14 @@ jobs:
         run: make proto-check-breaking PR_TARGET_REPO=https://github.com/${{ github.repository }}.git PR_TARGET_BRANCH=${{ github.base_ref }}
         working-directory: golang/cosmos
         if: env.GIT_DIFF
+  up-to-date:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: 'true'
+      - name: check protobuf definitions are up-to-date
```

You'll need Golang here:
```suggestion
      - uses: ./.github/actions/restore-golang
        with:
          go-version: '1.20'
      - name: check protobuf definitions are up-to-date
```